packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -43,7 +43,8 @@
     "bulletListTooltip": "Bulleted list",
     "numberListTooltip": "Numbered list",
     "increaseIndentTooltip": "Increase indent",
-    "decreaseIndentTooltip": "Decrease indent"
+    "decreaseIndentTooltip": "Decrease indent",
+    "richTextFormatButtonTooltip": "Format"
```

Comment:
Do we need richText prefix here?

Reply:
I was thinking about `formatButtonTooltip` first but it wasn't very clear what is format here so I went with `richTextFormatButtonTooltip`. Any suggestions?